.github/workflows/build-linux-installer-rpm.yml

```diff
@@ -203,6 +190,74 @@ jobs:
         name: chia-installers-linux-rpm-intel
         path: ${{ github.workspace }}/build_scripts/final_installer/
 
+    - name: Remove working files to exclude from cache
+      run: |
+        rm -rf ./chia-blockchain-gui/packages/gui/daemon
+
+  publish:
+    name: Publish amd64 RPM
+    runs-on: ubuntu-latest
+    needs:
+    - build
+    container:
+      image: chianetwork/rocky8-builder:latest
+    timeout-minutes: 5
+    strategy:
+      fail-fast: false
+      max-parallel: 4
```

Comment:
```suggestion
```
Same comment as before.. We should probably remove the other max-parallel in this file as well.